### DISCUSSION TITLE
Where is the Credential .env file located...?

### DISCUSSION THREAD

**Discussion by zluckytraveler** [Taxel/PlexTraktSync#887] [Support] [answered]
Im on macos, and I've looked everywhere for finding the `.env` file that holds the credentials, however I cannot find it no matter what. Where is it?

I've tried running plextraktsync via pipx and also docker, with a volume that I set to know the path, yet this file is still not there.

**Comment by glensc** [comment, marked as answer]
Use the `info` subcommand to find the config path.
